crates/nu-cli/src/commands/history/history_.rs

```diff
@@ -119,12 +123,18 @@ impl Command for History {
                                 create_history_record(idx, entry, long, head)
                             })
                         })
-                        .ok_or(ShellError::FileNotFound { span: head })?
+                        .ok_or(ShellError::FileNotFound {
+                            file: history_path.display().to_string(),
+                            span: head,
+                        })?
                         .into_pipeline_data(ctrlc)),
                 }
             }
         } else {
-            Err(ShellError::FileNotFound { span: head })
+            Err(ShellError::FileNotFound {
+                file: "history file".into(),
+                span: head,
+            })
```

Comment:
There will be a conflict with https://github.com/nushell/nushell/pull/11849 here. The changes from the other PR are the right one (this should change to a `ConfigDirNotFound` error)

Reply:
I think it's fine, we can get this changed later on that pr